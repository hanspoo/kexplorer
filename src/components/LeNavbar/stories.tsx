import { Story, Meta } from '@storybook/react'
import LeNavbar from '.'

export default {
  title: 'LeNavbar',
  component: LeNavbar
} as Meta

const items = [
  { label: 'Home', href: '#' },
  { label: 'Options', href: '#' }
]
export const Default: Story = () => (
  <LeNavbar navItems={items} title={'Primary Menu'} />
)
export const WithItemSelected: Story = () => (
  <LeNavbar navItems={items} title={'Primary Menu'} selected="Options" />
)
