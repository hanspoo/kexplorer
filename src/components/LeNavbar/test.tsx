import { render, screen } from '@testing-library/react'

import LeNavbar from '.'

describe('<LeNavbar />', () => {
  it('should render the heading', () => {
    const { container } = render(
      <LeNavbar title="Menu Importante" navItems={[]} />
    )

    expect(
      screen.getByRole('heading', { name: /Menu Importante/i })
    ).toBeInTheDocument()
  })
})
