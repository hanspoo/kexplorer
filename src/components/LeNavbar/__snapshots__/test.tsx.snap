// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LeNavbar /> should render the heading 1`] = `
<h1>
  LeNavbar
</h1>
`;
